Source/UI/NewUI/PaymentForm/Factory/Theme/BillingForm/Theme+BillingCountryInput.swift

```diff
@@ -0,0 +1,63 @@
+//
+//  Theme+BillingCountryInput.swift
+//  
+//
+//  Created by Alex Ioja-Yang on 10/08/2022.
+//
+
+import UIKit
+
+public extension Theme {
+    
+    /// Theme generated Billing Country Input Style
+    struct ThemeBillingCountryInput: CellButtonStyle {
```

Comment:
```suggestion
    struct BillingCountryInputStyle: CellButtonStyle {
```

Reply:
There is a bit of repetition on these suggestions so I will be ignoring next occurrences as I have addressed the naming format [here](https://github.com/checkout/frames-ios/pull/245#discussion_r943735159). Happy to keep the discussion there if you are not happy with my reasoning for this.